Neos.Neos/Classes/Service/HtmlAugmenter.php

```diff
@@ -69,9 +69,16 @@ protected function getHtmlRootElement($html)
         $domDocument = new \DOMDocument('1.0', 'UTF-8');
         // ignore parsing errors
         $useInternalErrorsBackup = libxml_use_internal_errors(true);
-        $domDocument->loadHTML($html);
-        $xPath = new \DOMXPath($domDocument);
-        $rootElement = $xPath->query('//html/body/*');
+
+        if ($domDocument->loadXML(html_entity_decode($html))) {
```

Comment:
I'm not sure that applying `html_entity_decode` on the HTML by default is the expected behavior. wouldn't that lead to incorrect results for sth like `&lt;div&gt;content&lt;/div&gt`?
I didn't understand why the decoding is required here - but if it really is it should maybe be applied at the *calling* side

Reply:
That looks weird indeed. We had so many problems with HMTL encoding/decoding on the UI side that a change like this must be well documented and tested in some projects to make sure we cover all bases.